common/src/transaction/cosmos_sdk/parser/structs/cosmos_raw_msg.rs

```diff
@@ -301,7 +303,7 @@ pub enum CosmosRawNormalMsg {
         /// the tokens to be transferred
         token: SingleCoin,
         /// Timeout height relative to the current block height.
-        /// The timeout is disabled when set to 0.
+        /// The timeout is disabled when set to Never.
```

Comment:
where `Never` declared?

Reply:
oh, that's a variant in `TimeoutHeight`, I switched back to height, but didn't updated the comment

Reply:
good catch!